### PR TITLE
Update latest encoding and ContentKeyPolicies json for Nov Stable rel…

### PR DESCRIPTION
**Pull request by giakas**
…ease

This release includes Fairplay Offline rental feature and Content aware encoding.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by anuchandy** [comment]
ARM spec, requesting reassign.

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/ContentKeyPolicies.json [hunk 1]
```diff
@@ -855,7 +882,7 @@
           "Lists all Content Key Policies": {
             "$ref": "examples/content-key-policies-list-all.json"
           },
-          "Lists Content Key Policies ordered by last modified.": {
+          "Lists Content Key Policies ordered by last modifed.": {
```
Comment (KrisBash) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/ContentKeyPolicies.json]: spelling

**Review by KrisBash — APPROVED** [review]
One comment regarding spelling

**Comment by yaohaizh** [comment]
@giakas  please address the review feedback and fix CI errors. Thanks.

**Comment by yaohaizh** [comment]
@giakas please fix model validation errors. Thanks.

**Pull request closed without merging**